calico-enterprise/multicluster/create-a-management-cluster/create-a-management-cluster-helm.mdx

```diff
@@ -66,37 +69,129 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
+  openssl req -new -key <(base64 -d my-managed-cluster.key.base64) -subj "/CN=my-managed-cluster" | \
+  openssl x509 -req -signkey <(base64 -d my-managed-cluster.key.base64) -days 365 | base64 -w 0 > my-managed-cluster.crt.base64
+  ```
+
+  Export the managed cluster variables:
+
+  ```bash
+  export MANAGED_CLUSTER_NAME=my-managed-cluster
+  export MANAGED_CLUSTER_OPERATOR_NAMESPACE=tigera-operator
+  export MANAGED_CLUSTER_CERTIFICATE=my-managed-cluster.crt.base64
+  ```
+
+1. Append the management cluster context to your `values.yaml`:
+
+  ```bash
+  cat "managementCluster:
+    enabled: true
+    address: $MANAGEMENT_CLUSTER_ADDR
+    service:
+      type: $EXT_SERVICE_TYPE
+      port: $EXT_SERVICE_PORT
+      targetPort: $EXT_SERVICE_TARGET_PORT
+      protocol: $EXT_SERVICE_PROTOCOL
+      nodePort: $EXT_SERVICE_NODE_PORT
+
+  managedClusters:
+    enabled: true
+    clusters:
+    - name: $MANAGED_CLUSTER_NAME
+      operatorNamespace: $MANAGED_CLUSTER_OPERATOR_NAMESPACE
+      certificate: $MANAGED_CLUSTER_CERTIFICATE" >> values.yaml
+  ```
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm 3 chart:
+
+  <CodeBlock language='bash'>
+     {'{{version}}' === 'master'
+       ? `helm install {{prodnamedash}} tigera/tigera-operator --version tigera-operator-v0.0 -f values.yaml \\
+--set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
+--set-file licenseKeyContent=<path/to/license/file/yaml> \\
+--namespace tigera-operator --create-namespace`
+       : `helm install {{prodnamedash}} tigera/tigera-operator --version tigera-operator-{{chart_version_name}} -f values.yaml \\
+--set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
+--set-file licenseKeyContent=<path/to/license/file/yaml> \\
+--namespace tigera-operator --create-namespace`}
+  </CodeBlock>
+
+1. You can now monitor progress with the following command:
+
+  ```bash
+  watch kubectl get tigerastatus
+  ```
+
+</TabItem>
+<TabItem label="LoadBalancer" value="LoadBalancer">
+
+To install a {{prodname}} [management](../create-a-management-cluster#value) cluster with Helm:
+
+1. [Configure a storage class for Calico Enterprise](../../operations/logstorage/create-storage).
+
+1. Export the service port number
+
+  ```bash
+  export EXT_SERVICE_PORT=9449
   ```
 
+  Export the public address or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>:$EXT_SERVICE_PORT
+  ```
+
+1. Export the service type, target port and protocol.
+  
+  ```bash
+  export EXT_SERVICE_TYPE=NodePort
+  export EXT_SERVICE_TARGET_PORT=9449
+  export EXT_SERVICE_PROTOCOL=tcp
+  ```
+  
   or if the external service is a LoadBalancer on AWS.
```

Comment:
The LoadBalancer tab item does not appear to be purely LoadBalancer instructions (see here and step 5 for example)